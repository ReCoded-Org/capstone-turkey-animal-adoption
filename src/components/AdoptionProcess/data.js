export let adoptionProcess = [
  {
    id: 1,
    img:
      "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
    text: "Find Your Pet",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
  {
    id: 2,
    img:
      "https://i.pinimg.com/736x/5f/80/85/5f80854fd1475958717a19e345695942.jpg",
    text: "Know Your Pet",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
  {
    id: 3,
    img:
      "https://i.pinimg.com/736x/2e/23/d3/2e23d3827c8293c27cea544a8acbcd59.jpg",
    text: "Take Your Pet Home",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
];
