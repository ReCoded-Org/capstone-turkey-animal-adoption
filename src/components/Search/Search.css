.searchContainer {
  justify-content: center;
}

.searchContainer h1 {
  color: #ff3e3e;
}

#adoptButton,
#adoptButton:hover {
  background: #ff7777;
  border: 1px solid #fff;
  padding: 0.375rem 0.75rem;
}

.firstDiv {
  text-align: start;
}

.secondDiv {
  text-align: end;
}

.pawIcon {
  margin-right: 0.5rem;
  transform: rotate(-20deg);
}

.searchForm {
  margin-top: 2rem;
}

.searchForm input,
.searchForm select {
  margin-right: 1rem;
  border: 2px solid #ff7777;
  border-radius: 30px;
  padding: 0.3rem 0.7rem;
  margin-bottom: 1rem;
}

.searchForm button {
  background: #fff;
  border: none;
  outline: none;
}

.searchIcon {
  color: #ff7777;
}

.searchCatLogo {
  max-width: 630px;
}

.loadPage h4,
.loadPage h6,
.searchForm select,
.searchForm option {
  color: #777777;
}

.adoptionForm input {
  width: 100%;
  margin-bottom: 1rem;
  padding: 0.5rem;
  border-radius: 10px;
  border: 2px solid #ff7777;
}

.searchForm input:focus,
.searchForm select:focus,
.adoptionForm input:focus {
  outline: none;
}

.adoptionFormModalBody h4 {
  color: #777777;
}

.modalFormBtn,
.modalFormBtn:hover,
.modalFormBtn:focus {
  background: #ff7777;
  border: 1px solid #fff;
}

.errorMessage {
  color: #3d3dfd;
  font-weight: bold;
  font-style: italic;
}

.cardCol {
  display: flex;
  justify-content: center;
}

/* Bootstrap Id */
#contained-modal-title-vcenter {
  color: #ff5555;
}
/* Bootstrap Id */

/* For Overriding */
#modalBodySearch {
  background: #ff7777;
}

#modalBodySearch p {
  color: #fff;
}

#cardText {
  display: flex;
  justify-content: space-between;
}
