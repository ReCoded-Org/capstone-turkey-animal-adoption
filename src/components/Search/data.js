export let searchGuests = {
    searchGuestsTitle: {
      title: "Adopt Your New Pet",
    },
    guestsResults: [
      {
        id: 1,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 5,
        breed: "Poodle",
        gender: "Female",
        name: "Sushi",
      },
      {
        id: 2,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 1,
        breed: "Terrier",
        gender: "Male",
        name: "Sushi",
      },
      {
        id: 3,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 2,
        breed: "Poodle",
        gender: "Female",
        name: "Sushi",
      },
      {
        id: 4,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 5,
        breed: "Poodle",
        gender: "Male",
        name: "Sushi",
      },
    ],
    locationCities : [
      "Istanbul",
      "Ankara",
      "Izmir"
    ],
    searchBreed : [
      "Poodle",
      "Terrier",
      "Rottweiler",
      "Golden Retriever",
      "Cat",
      "Tobby cat",
      "Scottish"
    ],
    searchAge : [
      "<3",
      ">3",
      "All"
    ],
    searchGender : [
      "Female",
      "Male",
    ],
  };