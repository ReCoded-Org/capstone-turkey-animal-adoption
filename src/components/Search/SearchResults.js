import React, { useState } from "react";
import { Col, Modal, Button, Card } from "react-bootstrap";
import "./Search.css";
import { BsGeoAlt } from "react-icons/bs";
import { FaPaw } from "react-icons/fa";

const SearchResults = ({ img, id, location, age, breed, gender }) => {
    const [show, setShow] = useState(false);

    const handleToggle = () => setShow(!show);

    return (
        <>
            <Col lg={4} className="mb-5">
                {/* <img src={img} alt="image" className="searchImage"/> */}
                <Card style={{ width: '18rem' }}>
                    <Card.Img variant="top" src={img} />
                    <Card.Body id="modalBody">
                        <Card.Text id="cardText">
                            <div className="firstDiv">
                                <p>{breed}</p>
                                <Button variant="primary" className="adoptButton" onClick={() => handleToggle()}>
                                    <FaPaw className="pawIcon"/>
                                    Adopt
                                </Button>
                            </div>
                            <div className="secondDiv">
                                <p>{gender}, {age}</p>
                                <p className="mt-4"><BsGeoAlt size={25}/> {location}</p>
                                {/* <BsGeoAlt size={25}/> */}
                            </div>
                        </Card.Text>
                    </Card.Body>
                </Card>
            </Col>
            <Modal show={show} onHide={() => handleToggle()} className="modal-open">
        <Modal.Header closeButton id="modalHeader">
          <Modal.Title>
          </Modal.Title>
        </Modal.Header>
        <Modal.Body id="modalBody">
          <p>My Name is</p>
          <p>I am old</p>
          <p>I am a</p>
        </Modal.Body>
        <Modal.Footer id="modalFooter">
          <Button
            className="shadow-none"
            id="closeButton"
            variant="secondary"
            onClick={() => handleToggle()}
          >
            Close
          </Button>
        </Modal.Footer>
      </Modal>
        </>
    );
};

export default SearchResults;