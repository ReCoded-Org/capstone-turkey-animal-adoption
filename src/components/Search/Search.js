import React, { useState } from "react";
import SearchResults from "./SearchResults";
import { Container, Row, Col, Button } from "react-bootstrap";
import { FaSearch } from "react-icons/fa";
import "./Search.css";
import { useFormik } from 'formik';

const Search = ({ searchGuests }) => {

  const [searchResults, setSearchresults] = React.useState([])

  const formik = useFormik({
    initialValues: {
      location: "",
      age: "",
      breed: "",
      gender: "",
    },
    onSubmit: values => {
      const searchItem = searchGuests.guestsResults.filter(item => {
        return (
          item.location
            .toLowerCase()
            .includes(
              values.location.toLowerCase()
            ) &&
          item.age
            .toLowerCase()
            .includes(
              values.age.toLowerCase()
            ) &&
          item.breed
            .toLowerCase()
            .includes(
              values.breed.toLowerCase()
            ) &&
          item.gender
            .toLowerCase()
            .includes(
              values.gender.toLowerCase()
            )
        )
      })
      return setSearchresults(searchItem)
      // return JSON.stringify(values, null, 2);
    },
  });

  const LoadingPage = () => {
    return (
      <Col sm={12} className="loadPage">
        <h4>Start Searching Now and Discover New Friends</h4>
        <h6>They are waiting for their new home</h6>
        <img src="https://timesofindia.indiatimes.com/photo/67586673.cms" className="w-100 searchCatLogo" />
      </Col>
    );
  };

  return (
    <Container>
      <Row className="searchContainer py-5">
        <h1>{searchGuests.searchGuestsTitle.title}</h1>
        <Col sm={12}>
          <form onSubmit={formik.handleSubmit} className="searchForm">
            <label htmlFor="location"></label>
            <select id="location" name="location" value={formik.values.location} {...formik.getFieldProps('location')}>
              <option value="Select Location">Select Location</option>
              {searchGuests.locationCities.map(city => <option>{city}</option>)}
            </select>

            <label htmlFor="age"></label>
            <input
              id="age"
              type="text"
              placeholder="age"
              {...formik.getFieldProps('age')}
            />

            <label htmlFor="breed"></label>
            <select id="breed" name="breed" value={formik.values.breed} {...formik.getFieldProps('breed')}>
              <option value="Select Breed">Select Breed</option>
              {searchGuests.searchBreed.map(breed => <option>{breed}</option>)}
            </select>

            <label htmlFor="gender"></label>
            <select name="gender" value={formik.values.gender} {...formik.getFieldProps('gender')}>
              <option value="Select Gender">Select Gender</option>
              <option value="Female">Female</option>
              <option value="Male">Male</option>
            </select>

            <button type="submit">
              <FaSearch size={27} className="searchIcon" />
            </button>
          </form>
        </Col>
        <Col sm={12}>
          <Row className="mt-5 items">
            {searchResults.length > 0 ? searchResults.map(result => {
              return (
                <SearchResults
                  key={result.id}
                  img={result.img}
                  location={result.location}
                  age={result.age}
                  breed={result.breed}
                  gender={result.gender}
                  name={result.name}
                />
              )
            }) : <LoadingPage />}
          </Row>
        </Col>
      </Row>
    </Container>
  );
};

export default Search;