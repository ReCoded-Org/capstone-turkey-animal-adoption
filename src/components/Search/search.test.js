import React from "react";
import renderer from "react-test-renderer";
import Search from "./Search.js";
import { searchGuests } from "./data";

it("renders Search correctly", () => {
  const tree = renderer.create(<Search searchGuests={searchGuests} />).toJSON();
  expect(tree).toMatchSnapshot();
});
