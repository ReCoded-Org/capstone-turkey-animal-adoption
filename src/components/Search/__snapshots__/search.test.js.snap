// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Search correctly 1`] = `
<div
  className="container"
>
  <div
    className="searchContainer py-5 row"
  >
    <h1>
      searchPage.title
    </h1>
    <div
      className="col-sm-12"
    >
      <form
        className="searchForm"
        onSubmit={[Function]}
      >
        <label
          htmlFor="location"
        />
        <select
          id="location"
          name="location"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value="Select Location"
          >
            searchPage.option0
          </option>
          <option>
            Istanbul
          </option>
          <option>
            Ankara
          </option>
          <option>
            Izmir
          </option>
          <option>
            Canakkale
          </option>
          <option>
            Antalya
          </option>
          <option>
            Alanya
          </option>
          <option>
            MuÄŸla
          </option>
          <option>
            Mersin
          </option>
        </select>
        <label
          htmlFor="age"
        />
        <select
          id="age"
          name="age"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value="Select Age"
          >
            searchPage.option1
          </option>
          <option>
            &lt;3
          </option>
          <option>
            &gt;3
          </option>
          <option>
            All
          </option>
        </select>
        <label
          htmlFor="breed"
        />
        <select
          id="breed"
          name="breed"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value="Select Breed"
          >
            searchPage.option2
          </option>
          <option>
            Poodle
          </option>
          <option>
            Terrier
          </option>
          <option>
            Rottweiler
          </option>
          <option>
            Golden Retriever
          </option>
          <option>
            Cat
          </option>
          <option>
            Tobby cat
          </option>
          <option>
            Scottish
          </option>
        </select>
        <label
          htmlFor="gender"
        />
        <select
          id="gender"
          name="gender"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            value="Select Gender"
          >
            searchPage.option3
          </option>
          <option>
            Female
          </option>
          <option>
            Male
          </option>
        </select>
        <button
          type="submit"
        >
          <svg
            className="searchIcon"
            fill="currentColor"
            height={27}
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width={27}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
            />
          </svg>
        </button>
      </form>
    </div>
    <div
      className="col-sm-12"
    >
      <div
        className="mt-5 items row"
      >
        <div
          className="mb-5 cardCol col-lg-4"
        >
          <div
            className="card"
            style={
              Object {
                "width": "18rem",
              }
            }
          >
            <img
              className="card-img-top"
              src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg"
            />
            <div
              className="card-body"
              id="modalBodySearch"
            >
              <div
                id="cardText"
              >
                <div
                  className="firstDiv"
                >
                  <p>
                    Poodle
                  </p>
                  <p>
                    Sushi
                  </p>
                  <button
                    className="shadow-none btn btn-primary"
                    disabled={false}
                    id="adoptButton"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="pawIcon"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"
                      />
                    </svg>
                    searchPage.button
                  </button>
                </div>
                <div
                  className="secondDiv"
                >
                  <p>
                    Female
                    , 
                    5
                  </p>
                  <p
                    className="mt-4"
                  >
                    <svg
                      fill="currentColor"
                      height={25}
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width={25}
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M8 16s6-5.686 6-10A6 6 0 002 6c0 4.314 6 10 6 10zm0-7a3 3 0 100-6 3 3 0 000 6z"
                        fillRule="evenodd"
                      />
                    </svg>
                     
                    Istanbul
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mb-5 cardCol col-lg-4"
        >
          <div
            className="card"
            style={
              Object {
                "width": "18rem",
              }
            }
          >
            <img
              className="card-img-top"
              src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg"
            />
            <div
              className="card-body"
              id="modalBodySearch"
            >
              <div
                id="cardText"
              >
                <div
                  className="firstDiv"
                >
                  <p>
                    Terrier
                  </p>
                  <p>
                    Sushi
                  </p>
                  <button
                    className="shadow-none btn btn-primary"
                    disabled={false}
                    id="adoptButton"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="pawIcon"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"
                      />
                    </svg>
                    searchPage.button
                  </button>
                </div>
                <div
                  className="secondDiv"
                >
                  <p>
                    Male
                    , 
                    1
                  </p>
                  <p
                    className="mt-4"
                  >
                    <svg
                      fill="currentColor"
                      height={25}
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width={25}
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M8 16s6-5.686 6-10A6 6 0 002 6c0 4.314 6 10 6 10zm0-7a3 3 0 100-6 3 3 0 000 6z"
                        fillRule="evenodd"
                      />
                    </svg>
                     
                    Istanbul
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mb-5 cardCol col-lg-4"
        >
          <div
            className="card"
            style={
              Object {
                "width": "18rem",
              }
            }
          >
            <img
              className="card-img-top"
              src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg"
            />
            <div
              className="card-body"
              id="modalBodySearch"
            >
              <div
                id="cardText"
              >
                <div
                  className="firstDiv"
                >
                  <p>
                    Poodle
                  </p>
                  <p>
                    Sushi
                  </p>
                  <button
                    className="shadow-none btn btn-primary"
                    disabled={false}
                    id="adoptButton"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="pawIcon"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"
                      />
                    </svg>
                    searchPage.button
                  </button>
                </div>
                <div
                  className="secondDiv"
                >
                  <p>
                    Female
                    , 
                    2
                  </p>
                  <p
                    className="mt-4"
                  >
                    <svg
                      fill="currentColor"
                      height={25}
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width={25}
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M8 16s6-5.686 6-10A6 6 0 002 6c0 4.314 6 10 6 10zm0-7a3 3 0 100-6 3 3 0 000 6z"
                        fillRule="evenodd"
                      />
                    </svg>
                     
                    Istanbul
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mb-5 cardCol col-lg-4"
        >
          <div
            className="card"
            style={
              Object {
                "width": "18rem",
              }
            }
          >
            <img
              className="card-img-top"
              src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg"
            />
            <div
              className="card-body"
              id="modalBodySearch"
            >
              <div
                id="cardText"
              >
                <div
                  className="firstDiv"
                >
                  <p>
                    Poodle
                  </p>
                  <p>
                    Sushi
                  </p>
                  <button
                    className="shadow-none btn btn-primary"
                    disabled={false}
                    id="adoptButton"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="pawIcon"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"
                      />
                    </svg>
                    searchPage.button
                  </button>
                </div>
                <div
                  className="secondDiv"
                >
                  <p>
                    Female
                    , 
                    5
                  </p>
                  <p
                    className="mt-4"
                  >
                    <svg
                      fill="currentColor"
                      height={25}
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width={25}
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M8 16s6-5.686 6-10A6 6 0 002 6c0 4.314 6 10 6 10zm0-7a3 3 0 100-6 3 3 0 000 6z"
                        fillRule="evenodd"
                      />
                    </svg>
                     
                    Istanbul
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mb-5 cardCol col-lg-4"
        >
          <div
            className="card"
            style={
              Object {
                "width": "18rem",
              }
            }
          >
            <img
              className="card-img-top"
              src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg"
            />
            <div
              className="card-body"
              id="modalBodySearch"
            >
              <div
                id="cardText"
              >
                <div
                  className="firstDiv"
                >
                  <p>
                    Rottweiler
                  </p>
                  <p>
                    Sushi
                  </p>
                  <button
                    className="shadow-none btn btn-primary"
                    disabled={false}
                    id="adoptButton"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="pawIcon"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"
                      />
                    </svg>
                    searchPage.button
                  </button>
                </div>
                <div
                  className="secondDiv"
                >
                  <p>
                    Male
                    , 
                    2
                  </p>
                  <p
                    className="mt-4"
                  >
                    <svg
                      fill="currentColor"
                      height={25}
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width={25}
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M8 16s6-5.686 6-10A6 6 0 002 6c0 4.314 6 10 6 10zm0-7a3 3 0 100-6 3 3 0 000 6z"
                        fillRule="evenodd"
                      />
                    </svg>
                     
                    Izmir
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mb-5 cardCol col-lg-4"
        >
          <div
            className="card"
            style={
              Object {
                "width": "18rem",
              }
            }
          >
            <img
              className="card-img-top"
              src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg"
            />
            <div
              className="card-body"
              id="modalBodySearch"
            >
              <div
                id="cardText"
              >
                <div
                  className="firstDiv"
                >
                  <p>
                    Poodle
                  </p>
                  <p>
                    Sushi
                  </p>
                  <button
                    className="shadow-none btn btn-primary"
                    disabled={false}
                    id="adoptButton"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      className="pawIcon"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"
                      />
                    </svg>
                    searchPage.button
                  </button>
                </div>
                <div
                  className="secondDiv"
                >
                  <p>
                    Male
                    , 
                    4
                  </p>
                  <p
                    className="mt-4"
                  >
                    <svg
                      fill="currentColor"
                      height={25}
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width={25}
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M8 16s6-5.686 6-10A6 6 0 002 6c0 4.314 6 10 6 10zm0-7a3 3 0 100-6 3 3 0 000 6z"
                        fillRule="evenodd"
                      />
                    </svg>
                     
                    Istanbul
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
